modules/codegen/test/src/smithy4s/codegen/internals/RendererSpec.scala

```diff
@@ -711,4 +711,69 @@ final class RendererSpec extends munit.ScalaCheckSuite {
     )
 
   }
+
+  test("nullable fields should be rendered as smithy4s.Nullable") {
+    val smithy =
+      """
+        |$version: "2.0"
+        |
+        |namespace smithy4s
+        |
+        |use alloy#nullable
+        |
+        |structure NullableExample {
+        |  @nullable
+        |  @required
+        |  nullableString: String
+        |}
+        |""".stripMargin
+
+    val contents = generateScalaCode(smithy).values
+    val definition = contents.find(_.contains("final case class NullableExample")).get
+
+    assert(
+      definition.contains("nullableString: Nullable[String]"),
```

Comment:
So, this test already passes before my changes to Renderer.scala and ToLine.scala. I don't understand this yet, because I don't know where else they would found and rendered as Nullable.

The other test is for sparse list and map types, which can have nullable elements. Those were previously rendered as Option.